src/framework/global/dataformatter.cpp

```diff
@@ -100,5 +100,5 @@ String DataFormatter::formatFileSize(size_t size)
     }
 
     //: Used to indicate file size. Ideally, keep the translation short; feel free to use an abbreviation.
-    return mtrc("global", "%n bytes", "", size);
+    return mtrc("global", "%n byte(s)", nullptr).arg(size);
```

Comment:
This is wrong. When working with `%n`, the number needs to be specified via the fourth argument of the mtrc function, and not via .arg, since that only works with %1 ... %9.